content/lessons/transacting/transacting-3/index.tsx

```diff
@@ -6,7 +6,7 @@ export default function Transacting3({ lang }) {
   const t = useTranslations(lang)
   return (
     <TerminalChallenge
-      expectedInput={'6a127461636f7320666f722065766572796f6e65'}
+      expectedInput={'7461636f7320666f722065766572796f6e65'}
```

Comment:
```suggestion
      expectedInput={'5361746f7368693433335061726b53744368696e6f43413933323433'}
```

Is there any way to take this value out and make it a constant somewhere? Needing to update it in two places doesn't leave me with a high level of confidence that I haven't missed another occurrence of it.

Reply:
I could use the first content page of the first challenge to be passed as a prop to `/content/lessons/transacting/transacting-3/index.tsx`

Reply:
Looking more into this, This is a one off because we are having both challenges in Transacting-2 and Transacting-3 use the same answer. But I do not think this will occur enough that we will need to create a template for this particular case